### ISSUE TITLE
结果中precision,recall,FB1全为0

### ISSUE THREAD

**Issue by cwj** [macanv/BERT-BiLSTM-CRF-NER#154]
你好，我下载代码后，对自己的训练数据进行标注，
并修改下面代码中self.labels一行
` # 通过读取train文件获取标签的方法会出现一定的风险。
        if os.path.exists(os.path.join(self.output_dir, 'label_list.pkl')):
            with codecs.open(os.path.join(self.output_dir, 'label_list.pkl'), 'rb') as rf:
                self.labels = pickle.load(rf)
        else:
            if len(self.labels) > 0:
                self.labels = self.labels.union(set(["X", "[CLS]", "[SEP]"]))
                with codecs.open(os.path.join(self.output_dir, 'label_list.pkl'), 'wb') as rf:
                    pickle.dump(self.labels, rf)
            else:
                self.labels = ["O", 'B-TIM', 'I-TIM', "B-PER", "I-PER", "B-ORG", "I-ORG", "B-LOC", "I-LOC", "X", "[CLS]", "[SEP]"]
        return self.labels`
改成自己标注，如下
self.labels = ["O", 'B-TECH', 'I-TECH', "E-TECH", "B-APP", "I-APP", "E-APP", "B-PRO", "I-PRO", "E-PRO", "X", "[CLS]", "[SEP]"]
我的数据标注是按照"O", 'B-TECH', 'I-TECH', "E-TECH", "B-APP", "I-APP", "E-APP", "B-PRO", "I-PRO", "E-PRO"标注的。
将run.py里的启动代码加到了bert_lstm_ner.py里，
启动代码bert_lstm_ner.py
如下：
nohup python3.6 /dir/BERT-BiLSTM-CRF-NER-master/bert_base/train/bert_lstm_ner.py -ner='ner' -data_dir=/dir/data/  -vocab_file=/dir/chinese_L-12_H-768_A-12/vocab.txt -bert_config_file=/dir/chinese_L-12_H-768_A-12/bert_config.json -init_checkpoint=/dir/chinese_L-12_H-768_A-12/bert_model.ckpt  -max_seq_length=128    -learning_rate=2e-5 -num_train_epochs=10.0 -output_dir=/dir/output/result_dir/ -batch_size=32 &
运行完后结果：
shape of input_ids (?, 128)
shape of input_ids (?, 128)
shape of input_ids (?, 128)
processed 2785945 tokens with 12273 phrases; found: 0 phrases; correct: 0.
accuracy:  98.34%; precision:   0.00%; recall:   0.00%; FB1:   0.00
              APP: precision:   0.00%; recall:   0.00%; FB1:   0.00  0
              PRO: precision:   0.00%; recall:   0.00%; FB1:   0.00  0
             TECH: precision:   0.00%; recall:   0.00%; FB1:   0.00  0
请问出现这个情况的可能原因是什么呢？还请不胜赐教，谢谢！

**Comment by Lyabo** [comment]
我也遇到同样的问题，请问你解决了吗？

**Comment by macanv** [comment]
BERT-BiLSTM-CRF-NER-master/bert_base/train/bert_lstm_ner.py
这个文件下，并没有可运行的代码。。。不知道你是怎么运行起来的。
如果你想使用python xx.py的形式训练，可以参考一下根目录下的run.py

**Comment by Lyabo** [comment]
有没有脚本可以直接调用训练好的模型呀？终端调用那个没有反应，不知该如何着手

**Comment by cwj** [comment]
> BERT-BiLSTM-CRF-NER-master/bert_base/train/bert_lstm_ner.py
> 这个文件下，并没有可运行的代码。。。不知道你是怎么运行起来的。
> 如果你想使用python xx.py的形式训练，可以参考一下根目录下的run.py

我已经说明了啊，将run.py里的启动代码加到了bert_lstm_ner.py里了。

**Comment by cwj** [comment]
> 我也遇到同样的问题，请问你解决了吗？

还没有解决呢

**Comment by cwj** [comment]
真是普大喜奔，出来的结果不全是0了
![image](https://user-images.githubusercontent.com/2813801/59811314-dc4e8e00-933b-11e9-953a-9f826d45b76d.png)
只是增加了数据量

**Comment by pengjiao123** [comment]
请问你的结果稳定吗？假如再增加一倍数据量
